app/views/static_pages/about.html.erb

```diff
@@ -0,0 +1,9 @@
+<% provide(:title, t("StaticPages.About.title")) %>
```

Comment:
[MUST] In ruby the convention should be lowercase and underscore
=> `static_pages.about.title`. Please fix all below i18n